_data/simulations/moose_psu_1d_STA/meta.yaml

```diff
@@ -0,0 +1,58 @@
+---
+  benchmark:
+    id: 1d
+    version: 1
+
+  metadata:
+    summary: Metadata for the Benchmark1 problems
+    timestamp: 02/20/2017
+    author: PC. Simon
+    email: pjs5523@psu.edu
+    timestamp: "Tue Feb  7 10:36:59 2017 -EST"
+    software:
+      name: moose
+      version: git commit 2593a61 on 2017-01-18
+    hardware:
+      computing plateform: Red Hat Entreprise Linux 6 (Cluster)
+      cores: 100
+      details:
+        - name: clock rate
+          values:
+            rate:
+            unit: GHz
+        - name: processor
+          values: ""
+    implementation:
+      repo:
+        url:
+        version:
```

Comment:
Also, a version is required. That can be the commit for the repository at the version that you ran the code.